README.md

```diff
@@ -29,10 +33,47 @@ bot will try to release on Fedora dist-git, on `master` branch and branches spec
 It should not create merge conflicts, but in case it does,
 you have to solve them first before attempting the release again.
 
+# Try it locally
+For now, the easiest way how to install release-bot is via pip.
+Other possible installations are through
+[Docker](#docker-image), [OpenShift](#openshift-template).
+```
+$ pip install release-bot
+``` 
+First interaction with release bot may be automated releases on Github. Let's do it. 
+
+#### 1. Create upstream repository 
+First of all, create a Github repository or use an existing one. This will be upstream repository
+were releases will be published. Within upstream repository create `release-conf.yaml` file which contains info on how to release the specific project.
+
+Copy and edit [release-conf.yaml](release-conf-example.yaml). For possible advanced setup check [the documentation for an upstream repository](#upstream-repository).
+
+#### 2. Create another private repository
```

Comment:
You don't need this when running the bot locally: that's actually specific to s2i and the deployment.

It's just fine to say that people need to have specific config files ready.